project/Build.scala

```diff
@@ -648,14 +649,39 @@ object Build {
           name := "Scala.js JS Envs",
           libraryDependencies ++= Seq(
               "io.apigee" % "rhino" % "1.7R5pre4",
-              "org.webjars" % "envjs" % "1.2",
-              "com.novocode" % "junit-interface" % "0.9" % "test"
+              "org.webjars" % "envjs" % "1.2"
           ) ++ ScalaJSPluginInternal.phantomJSJettyModules.map(_ % "provided"),
           previousArtifactSetting,
           binaryIssueFilters ++= BinaryIncompatibilities.JSEnvs
       )
   ).dependsOn(tools)
 
+  lazy val jsEnvsTestKit: Project = Project(
+      id = "jsEnvsTestKit",
+      base = file("js-envs-test-kit"),
+      settings = (
+          commonSettings ++ publishSettings ++ fatalWarningsSettings
+          ) ++ Seq(
+        name := "Scala.js JS Envs Test Kit",
+        libraryDependencies +=
+          "junit" % "junit" % "4.8.2",
+        previousArtifactSetting,
```

Comment:
As this is the first version with this package, it should be `None` for now. And we'll have to switch to `previousArtifactSetting` in the "Towards 0.6.10" commit.